model_utils.py

```diff
@@ -32,6 +35,18 @@ def get_local_huggingface_tokenizer_model(
     else:
         device_map = None
 
+    if quantize:
+        load_in_4bit = True
+        quantization_config = BitsAndBytesConfig(
+                load_in_4bit=True,
+                bnb_4bit_compute_dtype=torch.bfloat16,
+                bnb_4bit_use_double_quant=True,
+                bnb_4bit_quant_type='nf4'
+            )
```

Comment:
Small formatting suggestion:
```suggestion
        quantization_config = BitsAndBytesConfig(
            load_in_4bit=True,
            bnb_4bit_compute_dtype=torch.bfloat16,
            bnb_4bit_use_double_quant=True,
            bnb_4bit_quant_type='nf4'
        )
```

Reply:
thanks